src/scss/components/_donors.scss

```diff
@@ -0,0 +1,61 @@
+.donors_info {
+  @include grid;
+}
+
+.donors {
+  position: relative;
+  overflow-x: hidden;
+}
+
+.donors_image {
+  width: 100%;
+}
+
+.donors_description {
+  @include body-xl-med;
+
+  font-size: 2.5rem;
+  @include col(2, 8);
+}
+
+.donors_link_container {
+  @include link-typography;
+  @include col(2, 5);
+
+  display: flex;
+  padding: 2.5rem 0 19.5rem;
```

Comment:
I would suggest to use this bottom padding on the general section .donors

Reply:
Thanks for the comment, when I use it in .donors it does not allow me to handle the shape of the m